rust_src/src/character.rs

```diff
@@ -30,13 +30,11 @@ fn char_or_string_p(object: LispObject) -> LispObject {
 /// Convert the byte CH to multibyte character.
 #[lisp_fn]
 fn unibyte_char_to_multibyte(ch: LispObject) -> LispObject {
-    ch.is_character_or_error();
-    let mut c = ch.as_fixnum().unwrap() as u32;
+    let c = ch.as_character_or_error() as u32;
```

Comment:
The "as u32" should be unnecessary (especially since Codepoint is just an alias for it 😄)